README.md

```diff
@@ -51,8 +51,8 @@ Include JavaScript in html
 
 Sass import
 
-```sass
-@import 'src/scss/lazyframe'
+```scss
+@forward 'src/scss/lazyframe'
```

Comment:
Since you confirmed that including the styles in TS/JS is enough, this part is no longer needed.